core/Console.php

```diff
@@ -34,11 +34,23 @@ public function __construct()
 
         $this->getDefinition()->addOption($option);
 
+        $option = new InputOption('xhprof',
+            null,
+            InputOption::VALUE_NONE,
+            'Enable profiling with XHProf'
+        );
+
+        $this->getDefinition()->addOption($option);
```

Comment:
Not sure, we could log a warning and remove it once it becomes a problem. Eg someone might enable development mode on purpose to profile things uncached etc. Adding this logger would be a new issue I reckon as we would need to do this also when using `xhprof=1` in a URL.